src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -214,7 +214,7 @@ public static IEnumerable<UserDefinedFunction> CreateFunctions(IEnumerable<UDF>
                 Contracts.Assert(udf.IsParseValid);
 
                 var udfName = udf.Ident.Name;
-                if (texlFunctionSet.AnyWithName(udfName))
+                if (texlFunctionSet.AnyWithName(udfName) || nameResolver.Functions.WithName(udfName).Any(func => func is UserDefinedFunction))
```

Comment:
>.Any(func => func is UserDefinedFunction) [](http://example.com/codeflow?start=100&length=41)

Why this part?

Whether a function is a UDF or something else (builtin, reflectionFunction, etc) shouldn't impact our binding rules.

Reply:
this is to check for duplicates, If an UDF with same name is defined in the symbols , we error

Reply:
by why specifically UDF, as opposed to any derived TexlFunction with the same name conflict.